lib/subwayside_ui/gtfs.ex

```diff
@@ -0,0 +1,53 @@
+defmodule SubwaysideUi.GTFS do
+  @moduledoc """
+  Keeps GTFS-RT in-memory for providing published crowding information.
+  """
+  use GenServer
+  require Logger
+
+  @tick_interval 1_000
+
+  def start_link([]), do: GenServer.start_link(__MODULE__, nil, name: __MODULE__)
+
+  def init(_) do
+    tick()
+    {:ok, %{}}
+  end
+
+  defp tick, do: Process.send_after(self(), :tick, @tick_interval)
+
+  def handle_cast({:add, id}, tasks), do: {:noreply, MapSet.put(tasks, id)}
+
+  def handle_cast({:remove, id}, tasks), do: {:noreply, MapSet.delete(tasks, id)}
+
+  def handle_info(:tick, _gtfs) do
+    gtfs = download_gtfs()
+    tick()
+    {:noreply, gtfs}
+  end
+
+  def handle_info(_, gtfs), do: {:noreply, gtfs}
+
+  def download_gtfs do
+    result = get_url() |> HTTPoison.get!()
+
+    case result do
+      %HTTPoison.Response{body: body, status_code: 200} ->
+        deserialized_body =
+          body
+          |> Jason.decode!()
```

Comment:
suggestion(nitpick):
```suggestion
        deserialized_body = Jason.decode!(body)
```